src/encoded/static/scss/encoded/modules/_matrix.scss

```diff
@@ -15,6 +18,390 @@ $row-data-header-width: 200px;
     background: linear-gradient(90deg, rgba(0, 0, 0, 0.1), rgba(0, 0, 0, 0.0));
 }
 
+.epigenome-body-map-container {
+    border: 2px solid $Homo-sapiens;
+    position: relative;
+    .body-list-top {
+        border-top: 0;
+    }
+    &.expanded {
+        max-width: 675px;
+        position: absolute;
+        z-index: 99999;
+        background: white;
+        left: 0;
+        right: 0;
+        margin: auto;
+        top: 120px;
+        @media screen and (max-width: $screen-sm-min) {
+            width: 95%;
+        }
+    }
+    &.collapsed {
+        width: 0;
+        height: 0;
+        border: 0;
+        overflow: hidden;
+    }
+    .collapse-body-map {
+        cursor: pointer;
+        margin: 0;
+        padding: 0;
+        width: 100%;
+        font-size: 1rem;
+        border: 0;
+    }
+    .body-list {
+        .body-list-inner {
+            text-align: center;
+        }
+    }
+    .body-list-inner {
+        li {
+            display: inline;
+            &:after {
+                content: " ";
+                word-spacing: 0em;
```

Comment:
REQUIRED CHANGE. this should be: "word-space: 0;" I know it minor, but it causes a warning and technically its incorrect.